docs/introduction.rst

```diff
@@ -11,106 +11,143 @@ or modifying files produced by other programs (e.g.,
 `msprime <https://msprime.readthedocs.io/en/stable/>`_,
 `fwdpy11
 <https://fwdpy11.readthedocs.io/en/stable/pages/tsoverview.html>`_
-and `tsinfer <https://tsinfer.readthedocs.io/>`_) for use in SLiM. 
+and `tsinfer <   8555https://tsinfer.readthedocs.io/>`_) for use in SLiM. 
```

Comment:
I assume this was on accident?

Reply:
Oops! I was fixing a USB keyboard at the same time and didn't realise it was still plugged in to my laptop. 😆

Reply:
figured, but wondered if it was some advanced URL magic I didn't know about =)